### PR TITLE
add new script for opening issues on data purge

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

thoth-station/thoth-application#1644

## Description

Get all installations which use data which is being purged from the database and warn users by opening issues upstream.

### CODE REVIEW

File: create_purge_issues.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+# workflow-helpers
+# Copyright(C) 2020 Kevin Postlethwait
```
Comment (pacospace) [inline create_purge_issues.py:3]: 2021 :D

File: create_purge_issues.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+import os
+import logging
+
+from ogr.services.github import GithubService
+
+from thoth.storages import GraphDatabase
+from thoth.workflow_helpers import __service_version__
+
+g = GraphDatabase()
```
Comment (pacospace) [inline create_purge_issues.py:28]: don't you need to connect the datbase `g.connect()`

Comment (pacospace) [inline create_purge_issues.py:28]: should it be capital letter `GRAPH`?

Comment (KPostOffice) [reply create_purge_issues.py:28]: Yeah, it should be since it's a global constant. Thanks

File: create_purge_issues.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+"""This task creates issues in repositories using Kebechet GitHub app when data purges occur."""
+
+import os
+import logging
+
+from ogr.services.github import GithubService
+
+from thoth.storages import GraphDatabase
+from thoth.workflow_helpers import __service_version__
+
```
Comment (pacospace) [inline create_purge_issues.py:27]: we miss the metrics like in https://github.com/thoth-station/workflow-helpers/blob/5e1f52897be6980b22bcb2a173a5b278a6f931e0/graph_schema_update.py#L24 and I think it is ready!! Thanks

**Review by pacospace — COMMENTED** [review]
One last comment on the metrics

File: create_purge_issues.py (hunk 3 above)
Comment (KPostOffice) [reply create_purge_issues.py:27]: Does it need send_metrics if I haven't created any prometheus metrics to push to the gateway?

Comment (pacospace) [reply create_purge_issues.py:27]: Yes, those are standard metrics, for all workflow-helpers tasks using thoth-storages :)

**Review by pacospace — APPROVED** [review]

File: create_purge_issues.py (hunk 3 above)
Comment (pacospace) [reply create_purge_issues.py:27]: I wil add them in separate PR

**Pull request merged**